frontend/src/components/CodeEditor/index.tsx

```diff
@@ -66,8 +74,11 @@ const CodeEditor: React.FC<CodeEditorProps> = (props) => {
     }
 
     const loadTemplate = async () => {
-      await initDocument(uid, roomId, template);
-      setIsDocumentLoaded(true);
+      if (matchUser && partner && matchCriteria && questionId && questionTitle) {
+        await initDocument(uid, roomId, template, matchUser.id, partner.id, matchCriteria.language, questionId, questionTitle);
+        checkDocReady();
```

Comment:
ok i'll see if it works. also to reply ur question, i was thinking it might be better to make the detailed attempt private since it has their code and stuff

Reply:
oh is there a reason why the code has to be private

Reply:
not really, just a personal preference 😅